support-frontend/assets/components/directDebit/directDebitProgressiveDisclosure/components/playback.tsx

```diff
@@ -52,7 +60,16 @@ function Playback(props: {
 	sortCodeString: string;
 	buttonText: string;
 	allErrors: Array<Record<string, string>>;
+	setRecaptchaToken: (token: string) => void;
+	expireRecaptchaToken?: () => void;
+	recaptchaError: string;
 }): JSX.Element {
+	useRecaptchaV2(
+		recaptchaId,
+		props.setRecaptchaToken,
+		props.expireRecaptchaToken,
+	);
+
```

Comment:
We also moved this hook into the playback component so that we only call it when we're rendering the element the recaptcha will mount to